charts/feast/values.yaml

```diff
@@ -55,20 +55,24 @@ core:
 store:
  errors:
    type: "stdout"
- warehouse:
-   type: "bigquery"
-   # options: '{"project": "gcp-project-id", "dataset": "feast"}'
```

Comment:
I don't think we want to include changes to Helm config now, right?

Maybe this PR is just for sharing between us for the moment, that's fine. I don't know if the upstream project would want a `docker-compose.yml` contributed to the project or not since they'd probably encourage people to use Kubernetes + Helm locally instead, but if we do end up going with Feast then keeping it on our internal fork makes sense to me, for development convenience.

Reply:
When you say keeping this, did you mean the `docker-compose.yml` file, or the change in `values.yml` file? I'm a little confused by the conclusion whether I should keep this or remove? 
Or should I move this change to a different PR?

Reply:
Sorry, by "keeping it" I did mean the Docker Compose file. I think that should be on the "Agoda master" branch that we presumably maintain going forward (though I agree with what you mentioned in chat, we should make it build from the `Dockerfile`s in the repo instead of referencing the gcr.io registry images—`build: path/to/Dockerfile`).

If we're not going to be using Kubernetes for the time being then I think we shouldn't merge any changes to the Helm configuration, it's just going to be potential conflict churn to work around when syncing up with upstream updates.